polaris-tokens/src/themes/base/color.ts

```diff
@@ -970,7 +970,7 @@ export const color: {
     value: colorsExperimental.green[12],
   },
   'color-icon-critical': {
-    value: colorsExperimental.red[11],
+    value: colorsExperimental.red[12],
```

Comment:
```suggestion
    value: colorsExperimental.red[11],
```

We can actually keep this as `11`. We made some updates recently and I thought I'd caught the value references that changed but I missed this one. Thanks for calling it out!